CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
+Version 2.0.1 - January 10, 2020
+================================
+- Renamed plugin package name from airship to airship_flutter
+- Updated iOS SDK to 13.0.4
+- Updated Android SDK to 12.1.0
+- Implemented attributes functionality in iOS and Android
+
```

Comment:
How come no 2.0.0?

Reply:
wtf happened, I think I messed this up scripting or something

Reply:
good catch